oleesir/authors-haven
# chore__improveTestCoverage

**What does this Merge request do?**

Implements chore__improveTestCoverage

**Description of Tasks completed?**

Implements chore__improveTestCoverage

**How should this be manually tested?**

- git fetch on your local machine and check out to chore__improveTestCoverage branch
- Run "yarn test --coverage " to start the development server

**Any background context you want to provide?**
N/A


**Screenshots (if appropriate)**
XXX
